workflows/const.go

```diff
@@ -103,6 +104,42 @@ const (
 	ECSAMIKey              = "ecs.ami-id"
 )
 
+// EnvironmentTags used to set default tags in environment
+var EnvironmentTags = map[string]string{
```

Comment:
Please declare these as a struct so we can pass these around to later functions (see below)